util/errors/errors.go

```diff
@@ -93,6 +94,24 @@ func isTransientEtcdErr(err error) bool {
 	return false
 }
 
+func isTransientInterruptionErr(err error) bool {
+	// Handle transient node/resource related errors
+	if strings.Contains(err.Error(), "OOMKilled") {
```

Comment:
OOMKilled isn't always transient.

There is no way of removing items from this list for the setups or scenarios where it's not transient, so I'd like this removing.